sleap_nn/training/model_trainer.py

```
@@ -3,10 +3,12 @@
 from pathlib import Path
 from typing import Optional, List
 import functools
+import inspect
```

Ruff

`inspect` imported but unused

Remove unused import: `inspect`

(F401)